packages/react-components/react-card/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fluentui/react-card",
-  "version": "9.0.0-beta.47",
+  "version": "9.0.0-rc.0",
```

Comment:
you need to update the version also in react-components. that's why yarn failed as it tries to fetch this from npm - which is non existent

Reply:
Done!